account_bank_statement_import_paypal/models/account_bank_statement_import_paypal_parser.py

```diff
@@ -40,9 +40,8 @@ def parse(self, mapping, data_file, filename):
         ).name
         account_number = journal.bank_account_id.acc_number
 
-        name = _('%s: %s') % (
-            journal.code,
-            path.basename(filename),
+        name = _('[PayPal] %s') % (
```

Comment:
Why do the `name = _(...` twice? Isn't it enough to have it only below?

Reply:
This is because we have one return above if is not lines in csv.

If i calculated the full name above before first return i will need to copy another code statements that i use below in second calculation.

Reply:
missed that :)